a2d2-api/src/test/java/io/elimu/a2d2/processtest/GenericServiceTest.java

```diff
@@ -73,82 +73,14 @@ public void setUp() throws Exception {
 		deployManualDbKjar();
 	}
 	
-	private void add(String path, BaseResource resource, JarOutputStream target) throws IOException {
-	  BufferedInputStream in = null;
-	  try {
-		  byte[] content = resource.getBytes();
-		  JarEntry entry = new JarEntry(path);
-		  entry.setTime(System.currentTimeMillis() - 100_000);
-		  target.putNextEntry(entry);
-		  in = new BufferedInputStream(new ByteArrayInputStream(content));
-		  byte[] buffer = new byte[1024];
-		  while (true) {
-			  int count = in.read(buffer);
-			  if (count == -1) {
-				  break;
-			  }
-			  target.write(buffer, 0, count);
-		  }
-		  target.closeEntry();
-	  	} finally {
-	  		if (in != null) {
-	  			in.close();
-	  		}
-	  	}
-	}
+
 	
 	private void deployManualKjar() throws IOException {
-		deployKjar("generic-example-service.dsl", "1");
+		GenericTestUtils.deployKjar("generic-example-service.dsl", "1");
```

Comment:
@kevinliuelimu do we have a best practice for indentation - like the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s4.6-whitespace)? I see tabs in some source files and spaces in another. Let's just decide to be consistent.

Reply:
We should use tabs for indentation.